README.md

```diff
@@ -1,14 +1,36 @@
 
 # jpo-security-svcs
-This module expopsed a RESTful API for performing cryptographic functions. The following paths identify the functions:
+This module exposes a RESTful API for performing cryptographic functions. The cryptographic functions may be carried out against an on-prem instance, or against a remote instance. Note that for the remote signing use case, MTLS will be used to secure the communications. This requires some additional work on the part of configuration. The following paths identify the functions:
```

Comment:
This might flow a bit better if it was broken up between the description of the project and the addendum on cryptographic functions. Something like:

This module exposes a RESTful API for performing cryptographic functions.

The cryptographic functions may be carried out against an on-prem instance, or against a remote instance. Note that for the remote signing use case, MTLS will be used to secure the communications. This requires some additional work on the part of configuration. The following paths identify the functions:

Reply:
Tweaked it to pull the addendum out to a bottom paragraph

Reply:
I like it!